modules/@apostrophecms/command-menu/index.js

```diff
@@ -0,0 +1,163 @@
+const assert = require('assert').strict;
+const pipe = (...functions) => (initial) => functions.reduce((accumulator, current) => current(accumulator), initial);
+
+module.exports = {
+  options: {
+    alias: 'commandMenu'
+  },
+  handlers(self) {
+    return {
+      'apostrophe:ready': {
+        composeCommands() {
+          self.rawCommands = Object.values(self.apos.modules).flatMap(self.composeCommandsForModule);
+          self.removes = [];
+          self.commands = {};
+          self.groups = {};
+
+          const composed = pipe(self.composeCommand, self.composeRemove, self.composeGroup)({ rawCommands: self.rawCommands });
+          try {
+            Object.entries(composed.command).every(([ name, command ]) => self.validateCommand({ name, command }));
```

Comment:
You're using `every` but you don't seem to care about the return value. Use `forEach`, or fail in some appropriate way if you don't like the return value of `every`.

Reply:
I think you don't care about it because your asserts will stop the whole show anyway. If that's the case use forEach.